.npmignore

```diff
@@ -4,3 +4,7 @@ npm-debug.log
 coverage
 .gitignore
 .DS_Store
+.npmignore
+__tests__
```

Comment:
I'm not familiar with those release notes, can you link to them?

I use `npm explore` all the time, and I feel very strongly that tests should be distributed with packages.